_posts/2020/12/2020-12-30-release-notes.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+date: 2020-12-30
+title: "0.NN: Release Notes"
+author: olenagerasimova
+description: |
+  Key features and improvements description in the recent
+  version 0.NN, released just a few hours ago.
+keywords:
+  - artipie
+  - binary artifacts
+  - hosting of artifacts
+  - docker-adapter
+  - gem-adapter
+  - debian-adapter
+  - REST API
+---
+
+[v0.NN] was released on December NN. This month our two main directions were Docker-adapter and 
+management-api transfer. Features added to Docker-adapter:
+- Local [catalogs](https://github.com/artipie/docker-adapter/issues/374) and 
+[tags](https://github.com/artipie/docker-adapter/issues/373) listing
+- Support for mount blob endpoint in [remote](https://github.com/artipie/docker-adapter/issues/390) and 
+[local](https://github.com/artipie/docker-adapter/issues/394) repositories
+- Docker [catalog cache](https://github.com/artipie/docker-adapter/issues/367) implemented
+- Proxy [tags](https://github.com/artipie/docker-adapter/issues/375) support
```

Comment:
@olenagerasimova caching and proxy was added both for catalog and tags. I think it will be easier to say that catalogs and tags support was added for local and proxy repositories.